contrib/example-backend/options/cookie.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2022 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+type Cookie struct {
+	SigningKey    string
+	EncryptionKey string
+}
+
+func NewCookie() *Cookie {
+	return &Cookie{}
+}
+
+func (options *Cookie) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&options.SigningKey, "cookie-signing-key", options.SigningKey, "The key which is used to sign cookies, base64 encoded. Valid lengths are 32 or 64 bytes.")
```

Comment:
as what? base64?

Reply:
this should rather be a file for security, i.e. `cookie-signing-key-file`.

Reply:
this should rather be a file for security, i.e. `cookie-signing-key-file`.

Reply:
yes 👍 it's in the option help `The key which is used to sign cookies, base64 encoded.`

Reply:
> file for security

let's do this as a follow-up, there is prior art which uses CLI options. Having said that, I'd like to remove the cookie handling at all in a refactoring.